GDJS/GDJS/IDE/ExporterHelper.cpp

```diff
@@ -69,6 +69,23 @@ static void InsertUnique(std::vector<gd::String> &container, gd::String str) {
     container.push_back(str);
 }
 
+static gd::String CleanProjectName(gd::String projectName) {
+  gd::String partiallyCleanedProjectName = projectName;
+
+  static const gd::String allowedCharacters =
+      "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ ";
```

Comment:
What if people don't use this alphabet?

Reply:
Yep, I'll probably change the logic to be less strong and remove only the file system characters that are problematic.
Plus, if only those characters are used, use a default name.